plugins/auth-backend-module-pinniped-provider/src/authenticator.test.ts

```diff
@@ -143,6 +145,27 @@ describe('pinnipedAuthenticator', () => {
     };
   });
 
+  describe('#initialize', () => {
+    it('always returns a PinnipedStrategyFactory', async () => {
+      const pinnipedStrategyFactory = pinnipedAuthenticator.initialize({
+        callbackUrl: 'https://backstage.test/callback',
+        config: new ConfigReader({
+          federationDomain: 'https://federationDomain.test',
+          clientId: 'clientId',
+          clientSecret: 'clientSecret',
+        }),
+      });
+
+      const { providerStrategy, client } =
+        await pinnipedStrategyFactory.getStrategy();
```

Comment:
I think this might be redundant to the setup in the `beforeEach`?
```suggestion
      const { providerStrategy, client } =
        await authCtx.getStrategy();
```

Reply:
Refactored and changed.